### DISCUSSION TITLE
How do I download version 3.5 with pacman on arch

### DISCUSSION THREAD

**Discussion by GZLiew** [fish-shell/fish-shell#9081] [General]
brew and emerge is able to install the latest version 3.5 

when will other package manager get the latest version??

**Comment by faho** [comment]
Packages in the archlinux repositories are uploaded by the archlinux package maintainers, so they will be updated when the maintainer gets around to it.

Arch really doesn't like their maintainers being pestered about it, the best they allow you to do is to mark the package out-of-date on [the package site](https://archlinux.org/packages/community/x86_64/fish/), which it already has been since 3.5.0 came out a little over a month ago.

We have no influence at all on this. They don't want anyone to file bugs about a package being out of date.

So, you can do one of two things:

1. Wait for the maintainer
2. Grab the [source files](https://github.com/archlinux/svntogit-community/tree/packages/fish/trunk), update it to 3.5.1 and build it yourself. See https://wiki.archlinux.org/title/Creating_packages for more information.
